release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -109,6 +109,17 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see the xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[supported devices].
 
+[id="ocp-4-12-openshift-sdn-ovn-kubernetes-feature-migration-support"]
+==== OpenShift SDN to OVN-Kubernetes feature migration support
+
+If you migrate from the OpenShift SDN cluster network provider to the OVN-Kubernetes cluster network provider, you can migrate your configurations for the following capabilities:
```

Comment:
I can't put my finger on it, but starting with "If you migrate" then saying "you can migrate" later sounds a bit off to me. Just wondering if it should be "If you want to migrate from..." or "If you plan to migrate from..." at the beginning. Just thinking out loud :)